index.bs

```diff
@@ -4664,10 +4670,14 @@ The user agent must periodically run [=queue reports for delivery=] on the
 [=event-level report cache=] and [=aggregatable attribution report cache=].
 
 To <dfn>queue reports for delivery</dfn> given a [=set=] of
-[=attribution reports=] |cache|, run the following steps:
+[=attribution reports=] |cache| and a [=moment=] |now|, run the following steps:
 
 1. Let |reportsToSend| be a new [=list=].
 1. [=set/iterate|For each=] |report| of |cache|:
+    1. If |now| - |report|'s [=attribution report/trigger time=] is greater than
+        30 days:
+        1. [=set/Remove=] |report| from |cache|.
+        1. [=iteration/Continue=].
```

Comment:
optional: Remove this new step and instead prefix line 4689 with:

```
If |now| - |report|'s [=attribution report/trigger time=] is less than or equal to 30 days,
```

Reply:
I like it, done.